src/libraries/System.Drawing.Common/src/Resources/Strings.resx

```diff
@@ -461,4 +461,7 @@
   <data name="SystemDrawingCommon_PlatformNotSupported" xml:space="preserve">
     <value>System.Drawing.Common is not supported on this platform.</value>
   </data>
+  <data name="CachedBitmapNotSupported" xml:space="preserve">
+    <value>CachedBitmap is not supported on the current version of libgdiplus.</value>
```

Comment:
Add a hint that v6.1 is needed for this?
So a user could update libgdiplus and use that feature.